internal/pkg/service/buffer/storage/config.go

```diff
@@ -0,0 +1,40 @@
+package storage
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/staging"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/target"
+)
+
+// Config contains default configuration for the storage.
+// Default settings can be customized in definition.TableSink.
+type Config struct {
+	Local   local.Config   `configKey:"local"`
+	Staging staging.Config `configKey:"staging"`
+	Target  target.Config  `configKey:"target"`
+}
+
+// ConfigPatch is same as the Config, but with optional/nullable fields.
+// It is part of the definition.TableSink structure to allow modification of the default configuration.
+type ConfigPatch struct {
+	Local   local.ConfigPatch   `json:"local,omitempty"`
+	Staging staging.ConfigPatch `json:"staging,omitempty"`
+	Target  target.ConfigPatch  `json:"target,omitempty"`
+}
+
```

Comment:
`ConfigPatch` poskytuje moznost upravit globalnu konfiguraciu per `TableSink`.

Vacsina uzivatelov bude mat defaultne nastavenia, ale ak klient bude potrebovat napr. rychlejsi SSD disk a priplati si za to, tak to nebude poblem customizovat. Sieto nastavenia bude moct menit iba super-admin.